manifest.json

```diff
@@ -110,6 +110,7 @@
             "dev/5.6.7:5.6.7",
             "dev/5.6.6:5.6.6",
             "dev/5.6.4:5.6.4",
+            "dev/5.6.3:5.6.3",
```

Comment:
could you pls also add the play version, as we always have both dev and play version.

Reply:
Also play docker is for 5.6.3 now available